app/experimenter/nimbus-ui/src/components/PageEditBranches/FormBranches/index.tsx

```diff
@@ -116,6 +120,21 @@ export const FormBranches = ({
     dispatch({ type: "setFeatureConfig", value });
   };
 
+  const handleAddScreenshot = (branchIdx: number) => () => {
+    commitFormData();
+    dispatch({ type: "addScreenshotToBranch", branchIdx });
+  };
+
+  const handleRemoveScreenshot =
+    (branchIdx: number) => (screenshotIdx: number) => {
+      commitFormData();
+      dispatch({
+        type: "removeScreenshotFromBranch",
```

Comment:
Do we have constants somewhere for these action names?

Reply:
No variable constants defined, but they are constants within [a type defining all valid actions](https://github.com/mozilla/experimenter/blob/4deaedc748fad269e01ddbbd925142811c7969b6/app/experimenter/nimbus-ui/src/components/PageEditBranches/FormBranches/reducer/actions.ts#L43-L52). So, if any of the strings are typo'd, linting will fail.

Reply:
Oh interesting truuuueeee.... Okay cool that's good enough 👍

Reply:
Yeah, it's a little weird, but seems to be a convention to reduce some boilerplate and lean into TypeScript

Reply:
Yeah interesting I always think of it as `const SOMETHING = "whatever"` and then just refering to the variable so you never retype the string anywhere, but I have also done it this way with declaring the string literal right in the type declaration.  TypeScripts structural typing is super fancy and blows my mind every time I use it.  VSCode will probably autocomplete that string for you even.